content/_code-samples/build-a-wallet/desktop-js/view/4_tx-history.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8"/>
+    <meta http-equiv="Content-Security-Policy" content="default-src 'self'; script-src 'self'"/>
+    <meta http-equiv="X-Content-Security-Policy" content="default-src 'self'; script-src 'self'"/>
+    <title>XRPL Wallet Tutorial (JavaScript / Electron)</title>
+</head>
+
+<body>
+    <h3>Build a XRPL Wallet - Part 4/8</h3>
+
+    <fieldset>
+        <legend>Account</legend>
+        Classic Address: <strong id="account-address-classic"></strong><br/>
+        X-Address: <strong id="account-address-x"></strong><br/>
+        XRP Balance: <strong id="account-balance"></strong><br/>
+        XRP Reserved: <strong id="account-reserve"></strong><br/>
+    </fieldset>
+
+    <fieldset>
+        <legend>Latest validated ledger</legend>
+        Ledger Index: <strong id="ledger-index"></strong><br/>
+        Ledger Hash: <strong id="ledger-hash"></strong><br/>
+        Close Time: <strong id="ledger-close-time"></strong><br/>
+    </fieldset>
+
+    <fieldset>
+        <legend>Transactions:</legend>
+        <table id="tx-table">
+            <thead>
+            <tr>
+                <th>Confirmed</th>
+                <th>Type</th>
+                <th>From</th>
+                <th>To</th>
+                <th>Value Delivered</th>
+                <th>Hash</th>
+            </tr>
+            </thead>
+            <tbody></tbody>
+        </table>
+    </fieldset>
+
+    <dialog id="account-address-dialog">
+        <form method="dialog">
+            <div>
+                <label for="address-input">Enter account address:</label>
+                <input type="text" id="address-input" name="address-input" />
+            </div>
+            <div>
+                <button type="reset">Reset</button>
```

Comment:
This should be the submit button instead of the reset button (Same applies to all other instances of "Reset" in this PR I think. (There's at least 2 other spots it shows up when ctrl + F on the code view of the PR review). 

As of now, the `4_tx-history.template` fails to load the "Enter your address" pop up because it's expecting a submit, but there is none, which causes this step to not work because you can't populate the address.
```suggestion
                <button type="submit">Submit</button>
```